bedrock/firefox/firefox_details.py

```diff
@@ -53,6 +54,7 @@ class FirefoxDesktop(_ProductDetails):
     version_map = {
         'nightly': 'FIREFOX_NIGHTLY',
         'alpha': 'FIREFOX_AURORA',
```

Comment:
Is there a reason to keep this key/value? Should the value now be `FIREFOX_DEVEDITION`?

Reply:
I think so. Good catch.

Reply:
Turns out I can't remove this as some code still references "alpha", but I did need to change it to reference `FIREFOX_DEVEDITION` instead of `FIREFOX_AURORA`.